js/script.js

```diff
@@ -0,0 +1,27 @@
+import { $taskInput, $addButton, $taskList } from "./elements.js";
+import { showToastMessage } from "./utilities.js";
+
+let tasks = [];
+
+const addButtonHandler = () => {
+  const taskTitle = $taskInput.value.trim();
+  if (taskTitle) {
+    tasks.unshift(taskTitle);
+    const $tasksList = document.createElement("li");
+    $tasksList.textContent = taskTitle;
+    $taskList.prepend($tasksList);
```

Comment:
While this works, you'll face an issue in the future when you'll have to delete a node from the DOM. 

I suggest, for every change in `tasks` array item, we re construct the list from ground up.

```js

renderList()

function renderList(){
    $taskList.innerHTML = ""

    tasks.forEach(task => {
       const $task = createTask(task) // this will return a list node with all the necessary data and event listeners
       
       $taskList.prepend($task)
    })
}

```

Reply:
Bhai, I have implemented this function in next pr.